tbtaf/publisher/TBTAFPublisher.py

```diff
@@ -3,17 +3,22 @@
 
 @author: @andresmuro , @andres.alvarado , @mestradago , @rnunezc
 '''
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
In general, one  request on the design dimension. As you see all the bifucations on the implementation you can tell where all this is going. Therefore I request to implement a strategy pattern here in order to isolate the implementation specifics from TBTAFPublisher.

This way, TBTAFPublisher just calls a factory method which delivers a TBTAFReportGenerator from which we could have concrete implementations for HTML and PDF. This way TBTAF will just call the abstract methods but the specifics remain hidden at the concrete TBTAFReportGenerator implementation.

Reply:
Implemented in commit c96db212103859741c85616c38c01e8ca5527fbd